src/v/raft/tests/basic_raft_fixture_test.cc

```diff
@@ -211,122 +211,123 @@ FIXTURE_TEST(
  * This is possible as the protocol waits for the majority of nodes to
  * acknowledge receiving the message before making it visible.
  */
-TEST_F_CORO(
-  raft_fixture, validate_relaxed_consistency_visible_offset_advancement) {
```

Comment:
FYI you can skip tests to ensure it still compiles and doesn't bit rot via `GTEST_SKIP() << "Skipping test because ... 🙂";`

http://google.github.io/googletest/advanced.html#skipping-test-execution